tools/benchmark/src/Reporter.ts

```diff
@@ -131,11 +120,16 @@ export class BenchmarkReporter {
 			table.cell("status", `${pad(4)}${chalk.green("✔")}`);
 		}
 		table.cell("name", chalk.italic(testName));
+		table.cell("total time (s)", prettyNumber((result as BenchmarkData).elapsedSeconds, 2));
```

Comment:
Can you add a comment here to imply if we want to show any property that's not in customData should add here?